Compilation failure on previous version (2017.3)

Introduced by #2149

```
AppEngineGradlePluginFacetType.java:77: error: AppEngineGradlePluginFacetConfiguration is not abstract and does not override abstract method writeExternal(Element) in FacetConfiguration
  public static class AppEngineGradlePluginFacetConfiguration
                ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error
```

--- a/app-engine/java/src/com/google/cloud/tools/intellij/appengine/java/facet/standard/AppEngineGradlePluginFacetType.java
+++ b/app-engine/java/src/com/google/cloud/tools/intellij/appengine/java/facet/standard/AppEngineGradlePluginFacetType.java
@@ -29,7 +29,10 @@ import com.intellij.openapi.components.PersistentStateComponent;
 import com.intellij.openapi.module.JavaModuleType;
 import com.intellij.openapi.module.Module;
 import com.intellij.openapi.module.ModuleType;
+import com.intellij.openapi.util.InvalidDataException;
+import com.intellij.openapi.util.WriteExternalException;
 import javax.swing.Icon;
+import org.jdom.Element;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
@@ -84,6 +87,16 @@ public class AppEngineGradlePluginFacetType
     public FacetEditorTab[] createEditorTabs(
         FacetEditorContext editorContext, FacetValidatorsManager validatorsManager) {
       return new FacetEditorTab[0];
+    }
+
+    @Override
+    public void readExternal(Element element) throws InvalidDataException {
+      // Deprecated in favor of loadState
+    }
+
+    @Override
+    public void writeExternal(Element element) throws WriteExternalException {
+      // Deprecated in favor of getState
     }
 
     @Nullable

